app/middleware/meta.js

```diff
@@ -16,7 +16,9 @@ module.exports = options => {
     // try to support Keep-Alive Header
     const server = ctx.app.server;
     if (server && server.keepAliveTimeout && server.keepAliveTimeout >= 1000 && ctx.header.connection !== 'close') {
-      const timeout = parseInt(server.keepAliveTimeout / 1000);
```

Comment:
can you give a benchmark result?

Reply:
well, i need a little time for that ...

Reply:
if there is no significant difference, I think parseInt is much more readable

Reply:
Agree. I also know `readable` is better as usual projects, but **as a library, performance is better**, comments make up for the defects of `readable`.

Reply:
only if the performance difference is more valuable,  if not, perfect to wait for v8 engine upgrade.

Reply:
@dead-horse @atian25 
Here is the benchmark I just wrote for this.

## benchmark in the web

http://jsben.ch/bm3oa

## benchmark in node

https://runkit.com/embed/vysy4yl72jmj

The conclusion: ``~~number`` >> ``parseInt(number, 10)`` >> ``parseInt(number)``